docs/CONFIGURATION.md

```diff
@@ -44,3 +44,9 @@ OpenSALT uses the http://viima.github.io/jquery-comments/ bundle to allow editor
 OpenSALT can use Google's Firebase realtime database to update editors in real-time.
 
 Read the [docs/deployments/firebase.md](./deployments/firebase.md) file to see how to configure it.
+
+### Additional Fields
+
+For the purpose of increasing presentation appeal, OpenSALT allows for additional fields in release 2.2. To add additional fields to CfItem, navigate to /additionalfield and add a field with LsItem in the dropdown.
```

Comment:
Might want to say to "navigate to /additionalfield *as a Super User*"

Reply:
good idea, "... to /additionalfield as a logged in SuperUser"